src/h5web/providers/hdf5-models.ts

```diff
@@ -101,13 +101,18 @@ export enum HDF5TypeClass {
   VLen = 'H5T_VLEN',
   Compound = 'H5T_COMPOUND',
   Enum = 'H5T_ENUM',
+  Bool = 'H5T_BOOL',
 }
 
 export type HDF5Endianness = 'BE' | 'LE' | 'Native' | 'Not applicable';
 
+export interface HDF5BooleanType {
+  class: HDF5TypeClass.Bool;
+}
+
 export interface HDF5EnumType {
   class: HDF5TypeClass.Enum;
-  base: HDF5BaseType;
+  base: HDF5Type;
```

Comment:
As a added `HDF5BooleanType`, I realized that the distinction between `HDF5BaseType` and `HDF5AdvancedType` is our doing and has nothing to do with the `base` of advanced types. 

Perhaps we should rename it `HDF5BasicType` to avoid confusion ?

Reply:
Up to you, I'm fine with both.

Reply:
I will leave it now and remove it when working on #310